api/src/main/java/org/openmrs/module/appointments/service/impl/RecurringAppointmentServiceImpl.java

```diff
@@ -108,7 +107,7 @@ public void setEditAppointmentValidators(List<AppointmentValidator> editAppointm
     public void changeStatus(Appointment appointment, String status, Date onDate, String clientTimeZone) {
```

Comment:
Should have notes in audit in a different way. 
"String notes = onDate != null ? onDate.toInstant().toString(): null;" - This is how it is AppointmentsServiceImpl